packages/prop-controllers/src/data.ts

```diff
@@ -63,3 +66,10 @@ export const responsiveNumberValueSchema = createResponsiveValueSchema(
 )
 
 export type ResponsiveNumberValue = z.infer<typeof responsiveNumberValueSchema>
+
+export const responsiveSelectValueSchema = createResponsiveValueSchema(
+  z.string(),
+)
```

Comment:
Since the select prop controller is generic over options, would it make sense to have a `createResponsiveSelectValueSchema` that takes in options?

Reply:
Right now we are using schemas only to validate values in prop controllers that are created based on deserialized control definitions, and the finer-typed options do not survive control serialization.